scss/xy-grid/_classes.scss

```diff
@@ -127,15 +127,20 @@
   }
 }
 
-// Margin Grid classes
+/// Margin Grid classes
+///
+/// @param {List} $gutter-position [left right] - The position to apply gutters to. Accepts `top`, `bottom`, `left`, `right` in any combination.
+/// @param {Boolean} $vertical [false] - Direction of the gutters to output. See `$gutter-position`.
```

Comment:
Shouldn't this be a List too?

Reply:
I think both is correct, but boolean describe it better for $vertical

Reply:
boolean = true / false. This is not a direction (string).

Reply:
> $vertical [false] - Direction of the gutters to output. See ...

makes not much sense.